Description: Adapt to libapt-pkg7.0 ABI
 flNotDir() returns a string_view now, convert it to an std::string
 so it can be stored.
Author: Julian Andres Klode <jak@debian.org>

--- rust-rust-apt-0.7.0.orig/apt-pkg-c/cache.h
+++ rust-rust-apt-0.7.0/apt-pkg-c/cache.h
@@ -70,7 +70,7 @@ inline rust::Vec<SourceURI> Cache::sourc
 	ptr->GetSourceList()->GetIndexes(&fetcher, true);
 	pkgAcquire::UriIterator I = fetcher.UriBegin();
 	for (; I != fetcher.UriEnd(); ++I) {
-		list.push_back(SourceURI{I->URI, flNotDir(I->Owner->DestFile)});
+		list.push_back(SourceURI{I->URI, std::string{flNotDir(I->Owner->DestFile)}});
 	}
 	return list;
 }
